apps/sms/services/js/messaging/messaging_service.js

```diff
@@ -61,7 +63,11 @@ var MessagingService = {
 
     this.initService();
 
-    this.mozMobileMessageClient = bridge.client(MOBILE_MESSAGE_CLIENT_NAME);
+    this.mozMobileMessageClient = bridge.client({
+      service: MOBILE_MESSAGE_CLIENT_NAME,
+      endpoint: new BroadcastChannel(MOBILE_MESSAGE_CLIENT_NAME + '-channel'),
```

Comment:
yes I think we need a bug and a TODO with the bug number here. This is not that important for retrieveMMS but will be for other cases :/